labkey/domain.py

```diff
@@ -109,6 +109,7 @@ def __init__(self, **kwargs):
         )
         self.type_editable = kwargs.pop("type_editable", kwargs.pop("typeEditable", None))
         self.url = kwargs.pop("url", None)
+        self.value_expression = kwargs.pop("value_expression", kwargs.pop("valueExpression", None))
```

Comment:
I really dislike that we support `snake_case` and `camelCase` variable names here, but since we do it for others it's probably the right choice.